src/reanimated2/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { RefObject } from 'react';
+import { measure } from '.';
+
+export interface ComponentCoords {
+  x: number;
+  y: number;
+}
+
+export function convertCoords<T>(
```

Comment:
I think that `getRelativeCoords` is a better name for this function because is more descriptive.